lib/assets/stylesheets/chef/_settings.scss

```diff
@@ -16,3 +16,5 @@ $logo-width-vertical:             76px;
 $logo-height-vertical:            82px;
 $logo-width-horizontal:           256px;
 $logo-height-horizontal:          60px;
+
+$chef-type-label:                   Montserrat, "Helvetica Neue", Helvetica, Arial, sans-serif !default;
```

Comment:
We should take out this line. The label's `!default` is set in `themes/_default`, and if a visual component needs something that's defined in a theme, it should include a reference to the theme. See [logo.scss](https://github.com/chef/chef-web-core/blob/master/lib/assets/stylesheets/chef/components/logo.scss#L95-L96), for example. Adding this below the `settings` include to icon.scss fixes things up: 

```
@import '../themes/default';
``` 

This gives the component a sensible default based on a common theme, but makes it overridable by using `!default`.